src/core/client/stream/tabs/Comments/Comment/ReplyCommentForm/ReplyCommentForm.tsx

```diff
@@ -57,6 +57,7 @@ const ReplyCommentForm: FunctionComponent<ReplyCommentFormProps> = props => {
               name="body"
               validate={getCommentBodyValidators(props.min, props.max)}
             >
+              {/* FIXME: (wyattjoh) reorganize this */}
```

Comment:
Was this left in as a personal reminder, or as a "defer util later" notice?

Reply:
`CORL-492`